src/datetime/mod.rs

```diff
@@ -604,6 +604,25 @@ impl DateTime<FixedOffset> {
         parse(&mut parsed, s, StrftimeItems::new(fmt))?;
         parsed.to_datetime()
     }
+
+    /// Parses a string from a user-specified format into a `DateTime<FixedOffset>` value, and a
+    /// slice with the remaining portion of the string.
+    ///
+    /// Note that this method *requires a timezone* in the input string. See
+    /// [`NaiveDateTime::parse_from_str`](./naive/struct.NaiveDateTime.html#method.parse_from_str)
+    /// for a version that does not require a timezone in the to-be-parsed str. The returned
+    /// [`DateTime`] value will have a [`FixedOffset`] reflecting the parsed timezone.
+    ///
+    /// See the [`format::strftime` module](./format/strftime/index.html) for supported format
+    /// sequences.
+    pub fn parse_and_remainder<'a>(
```

Comment:
Please add `#[must_use]` where appropriate.  I think all of these new functions need that.

There is a closed PR on `0.5` branch that adds `#[must_use]` so this gets ahead of some extra work. https://github.com/chronotope/chrono/pull/1007

Reply:
All these methods return a `ParseResult` which already gives these warnings.